src/components/VideoPlayer/index.js

```diff
@@ -89,6 +89,19 @@ export default class VideoPlayer extends PureComponent {
     }
   }
 
+  hideOpenScreens = () => {
```

Comment:
I made a separate function because I also wanted to hide all screens on seeking (not just playing), which would mean repeating it on line 116 and again on line 127